### ISSUE TITLE
Not building with php enabled

### ISSUE THREAD

**Issue by ThauEx** [unbit/uwsgi#1672]
Original issue: https://github.com/Homebrew/homebrew-core/issues/19874

When I try to build uwsgi with php on macOS 10.13 (via homebrew), it's failing.

The full log is here:
https://gist.github.com/ThauEx/71ddcf56d8cb368a8c85a5fe5ee7e8eb
and the one for the php module itself is this one: https://gist.githubusercontent.com/ThauEx/71ddcf56d8cb368a8c85a5fe5ee7e8eb/raw/e1210d76eefeea0df7c1bef6b776a48f3222c024/65.python

**Comment by xrmx** [comment]
TLDR;
```
ld: library not found for -lcrypto.35
```

**Comment by ThauEx** [comment]
True, but it tries to include something from the internal sdk (`MacOSX10.13.Internal.sdk/usr/local/libressl/lib`) and I think this is at least one of the issues...
